tests/conftest.py

```diff
@@ -105,13 +105,13 @@ def compileSolidity(relativeFilePath):
                 }
             },
             'settings': {
-                'remappings': [ 'ROOT=/augur-core/src' ]
+                'remappings': [ 'ROOT=%' % resolveRelativePath("/augur-core/src") ]
```

Comment:
I'll look at this and change possibly. I see why this seems odd but it does actually work though.